### ISSUE TITLE
[CLOSED] Add property support to entity

### ISSUE THREAD

**Issue by twogee** [antlibs/ant-http#22]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Monday May 11, 2015 at 22:05 GMT_
_Originally opened as https://github.com/twogee/missing-link/issues/22_

---

```
It would be nice to be able to retrieve the contents of an entity from a 
property, e.g. <entity property="artifactory.build.info"/>.

I have implemented this in a local copy but as I have incorporated in the patch 
in issue 13, a patch would not be that short and sweet. Anyway, it is trivial 
to implement this.
```

Original issue reported on code.google.com by `anders.j...@gmail.com` on 16 Aug 2011 at 11:22

**Comment by twogee** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Monday May 11, 2015 at 22:05 GMT_

---

Original comment by `alex.she...@gmail.com` on 18 Aug 2011 at 12:30
- Changed state: **Accepted**
- Added labels: **Milestone-1.1.3**, **OpSys-All**, **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by twogee** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Monday May 11, 2015 at 22:05 GMT_

---

```
Added a "value" attribute to the entity node, which takes precedence after the 
file parameter but before textual content nested inside <entity>..</entity>.  
This is a simple String property, ant takes care of expanding properties 
automatically in this scenario.  

Example:  value="Hello World" or value="${my.prop}"

Will be available in 1.1.3
```

Original comment by `alex.she...@gmail.com` on 18 Aug 2011 at 12:54
- Changed state: **Fixed**

**Comment by twogee** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Monday May 11, 2015 at 22:05 GMT_

---

Original comment by `alex.she...@gmail.com` on 19 Aug 2011 at 12:51
- Added labels: **Project-ant-http**
